Validation/CTPPS/python/ctppsHepMCDistributionPlotter_cfi.py

```diff
@@ -0,0 +1,8 @@
+from Validation.CTPPS.CTPPSHepMCDistributionPlotterDefault_cfi import CTPPSHepMCDistributionPlotterDefault as _CTPPSHepMCDistributionPlotterDefault
+CTPPSHepMCDistributionPlotter = _CTPPSHepMCDistributionPlotterDefault.clone()
+
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(CTPPSHepMCDistributionPlotter, useNewLHCInfo = True)
+
+from Configuration.Eras.Modifier_ctpps_directSim_cff import ctpps_directSim
+ctpps_directSim.toModify(CTPPSHepMCDistributionPlotter, useNewLHCInfo = False)
```

Comment:
All these could be written everywhere in a more compact way as
```suggestion
from Configuration.Eras.Modifier_run3_common_cff import run3_common
from Configuration.Eras.Modifier_ctpps_directSim_cff import ctpps_directSim
(run3_common & ~ctpps_directSim).toModify(CTPPSHepMCDistributionPlotter, useNewLHCInfo = True)
```

but I don't know if it suits your needs: take it as a possible suggestion, not as a request.

Reply:
Well I do admit it is more compact but definitely reduces code clarity - at least in my opinion. Using `~` symbol makes the code harder to understand and misreading just one symbol completety inverts the meaning of the code :P